<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="Menus">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="LoginorRegister">
     <widget class="QTextEdit" name="UsernameText">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>280</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>8514oem</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QGraphicsView" name="UserName">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="Password">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>320</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="PasswordTextEdit" name="PasswordText">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>320</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>8514oem</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="HoverButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>400</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="HoverButton" name="GuestButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>550</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="MainMenu">
     <widget class="HoverButton" name="PlayButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>270</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="HoverButton" name="StatsButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>370</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="HoverButton" name="ExitButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>470</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PlayMode">
     <widget class="HoverButton" name="VsPlayer">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>270</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="HoverButton" name="VsAI">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>430</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="StatScreen"/>
    <widget class="QWidget" name="Arena">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <widget class="QPushButton" name="index3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 5px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="index5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 5px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="index1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 5px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="index8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 5px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="index4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 5px solid white;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="index7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 5px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="index2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 5px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="index6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 5px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="index0">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
border: 5px solid white;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="ResultsScreen"/>
    <widget class="QWidget" name="UnUsed"/>
   </widget>
   <widget class="QGraphicsView" name="MainImage">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>30</y>
      <width>480</width>
      <height>192</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
    <property name="rubberBandSelectionMode">
     <enum>Qt::ItemSelectionMode::IntersectsItemShape</enum>
    </property>
   </widget>
   <widget class="QLabel" name="welcometext">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>210</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome Back:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>210</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordTextEdit</class>
   <extends>QTextEdit</extends>
   <header>PasswordTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>HoverButton</class>
   <extends>QPushButton</extends>
   <header>hoverbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
